examples/features/broker-balancer/evenly-redirect/readme.md

```diff
@@ -0,0 +1,6 @@
+# Evenly Redirect Example
+
+To run the example, simply type **mvn verify** from this directory, or **mvn -PnoServer verify** if you want to create and start the broker manually.
+
+This example demonstrates how incoming client connections are evenly redirected across two brokers
+using a third broker with a broker balancer to redirect incoming client connections.
```

Comment:
yes I would need a persistent cache to get Durable Subscription after a restart

Reply:
Yep, and I think its fine not having that, just thought it might be worth noting (maybe in the code comment instead) as the caching may catch some folks out if they dont consider that.

Reply:
doc updated